app/controllers/admin/backups_controller.rb

```diff
@@ -111,10 +111,10 @@ def token_reset_flash_message(token)
     ]
   end
 
-  def token_reset_failed!(e)
-    Rails.logger.error "Failed to reset user ##{current_user.id}'s Backup key: #{e}"
+  def token_reset_failed!(key)
```

Comment:
This is an exception which is expected here, hence `e`. I think it's rather the log message that's wrong in saying "Backup key:". So I'd just remove the word 'key' from the log message and keep the variable name.

If you look further below you can see that `e.message` is used so we definitely have an exception here.

Reply:
fixed in 4f701c8d48